repos/system_upgrade/el7toel8/actors/kernel/checkinstalleddevelkernels/libraries/library.py

```diff
@@ -0,0 +1,28 @@
+from leapp import reporting
+from leapp.libraries.stdlib import api
+from leapp.models import InstalledRedHatSignedRPM
+
+
+def get_kernel_devel_rpms():
+    rpms = next(api.consume(InstalledRedHatSignedRPM), InstalledRedHatSignedRPM())
+    return [pkg for pkg in rpms.items if pkg.name == 'kernel-devel'].sort(key=lambda pkg: pkg.version)
```

Comment:
versions are same. you should focus on release

Reply:
Ah, thanks. I confused the fields. :|